godtools/App/Share/Data/ResourcesRepository/Cache/Sync/RealmResourcesCacheSync.swift

```diff
@@ -0,0 +1,192 @@
+//
+//  RealmResourcesCacheSync.swift
+//  godtools
+//
+//  Created by Levi Eggert on 7/26/22.
+//  Copyright © 2022 Cru. All rights reserved.
+//
+
+import Foundation
+import RealmSwift
+import Combine
+
+class RealmResourcesCacheSync {
+    
+    typealias SHA256PlusPathExtension = String
+    typealias ResourceId = String
+    typealias TranslationId = String
+    
+    private let realmDatabase: RealmDatabase
+    private let resourcesSyncedNotificationName = Notification.Name("resourcesCache.notification.resourcesSynced")
+    
+    required init(realmDatabase: RealmDatabase) {
+        
+        self.realmDatabase = realmDatabase
+    }
+    
+    func getResourcesSyncedPublisher() -> NotificationCenter.Publisher {
+        NotificationCenter.default.publisher(for: resourcesSyncedNotificationName)
+    }
+    
+    func syncResources(languagesSyncResult: RealmLanguagesCacheSyncResult, resourcesPlusLatestTranslationsAndAttachments: ResourcesPlusLatestTranslationsAndAttachmentsModel) -> AnyPublisher<RealmResourcesCacheSyncResult, Error> {
```

Comment:
Is this class supposed to be removed since `InitialDataDownloaderResourcesSync` appears to do the same thing?  Or is this one of the things you're coming back to later?

Reply:
Hey @rachaelblue I'm going to be removing ```InitialDataDownloaderResourcesSync```.  ResourcesRepository will sync the resources and latest attachments and latest translations.  LanguagesRepository will sync the languages.
Then InitialDataDownloader will depend on those 2 classes to kick off syncing of those objects, it will also kick off any other initial downloading for translations and attachments.